apps/devhub/tests/test_views.py

```diff
@@ -1207,6 +1212,23 @@ def test_delete_and_recreate_credentials(self):
         assert new_key.key != old_key.key
         assert new_key.secret != old_key.secret
 
+    def test_retreive_credentials(self):
+        client = Client(enforce_csrf_checks=True)
```

Comment:
I'm confused about this. I assumed that enforcing the csrf checks would cause your GET a few lines down to fail because it's not passing a CSRF token in the header. What I wanted to see was a test for this kind of 403 response, not a 200 response. It might be 200'ing if the test client disables all middleware?